serialize.go

```diff
@@ -21,12 +27,15 @@ func SaveToFile(j *Jar, path string) error {
 	return j.WriteTo(f)
```

Comment:
If we have multiple clients on the same machine, wouldn't they override the content?